src/dyad/core/CMakeLists.txt

```diff
@@ -21,8 +21,8 @@ set(DYAD_CTX_PRIVATE_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/dyad_ctx.h
 
 add_library(${PROJECT_NAME}_core SHARED ${DYAD_CORE_SRC}
             ${DYAD_CORE_PUBLIC_HEADERS} ${DYAD_CORE_PRIVATE_HEADERS})
-set_target_properties(${PROJECT_NAME}_core PROPERTIES CMAKE_INSTALL_RPATH
-                      "${CMAKE_INSTALL_PREFIX}/${DYAD_LIBDIR}")
+# set_target_properties(${PROJECT_NAME}_core PROPERTIES CMAKE_INSTALL_RPATH
```

Comment:
@ilumsden Why did we comment this.

Reply:
They were leftover, commented-out versions of replaced logic. They've been removed.